tools/packaging/tests/Dockerfile/FedoraDockerfile.in

```diff
@@ -14,15 +14,14 @@ ENV GOPATH=/home/go
 ENV TESTS_REPOSITORY_PATH="${GOPATH}/src/${TESTS_REPO}"
 ENV AGENT_INIT=yes TEST_INITRD=yes OSBUILDER_DISTRO=alpine
 
-# Install packages
-RUN sudo dnf -y install kata-proxy kata-ksm-throttler kata-osbuilder kata-runtime kata-shim
-RUN sudo mkdir "${GOPATH}"
-RUN sudo dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
-RUN sudo dnf makecache
-RUN sudo dnf -y install docker-ce
-RUN go get -d "${TESTS_REPO}"
-RUN cd "${TESTS_REPOSITORY_PATH}" && .ci/install_kata_image.sh
-RUN cd "${TESTS_REPOSITORY_PATH}" && .ci/install_kata_kernel.sh
-RUN kata-runtime kata-env
+# Install packages and build and install Kata Containers
+RUN dnf -y install kata-proxy kata-ksm-throttler kata-osbuilder kata-runtime kata-shim && \
+    mkdir "${GOPATH}" && \
+    dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo && \
+    dnf makecache && dnf -y install docker-ce && dnf clean all && \
+    go get -d "${TESTS_REPO}" && \
```

Comment:
We can change "go get -d" to "git clone --depth=1" for reducing the size of the repo and increasing the building speed